### PR TITLE
Vite AEP Web Sandbox

### PR DESCRIPTION
**Pull request by shammowla**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [ ] I have made any necessary test changes and all tests pass.
- [ ] I have run the Sandbox successfully.

### CODE REVIEW

File: vite-sandbox/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>AEP Web SDK Sandbox - Home Page</title>
+    <script>
+      !function(n,o){o.forEach(function(o){n[o]||((n.__alloyNS=n.__alloyNS||
+      []).push(o),n[o]=function(){var u=arguments;return new Promise(
+      function(i,l){n[o].q.push([i,l,u])})},n[o].q=[])})}
+      (window,["alloy"]);
+    </script>
+    <script src="https://cdn1.adoberesources.net/alloy/2.7.0/alloy.js" async></script>
```
Comment (jfkhoury) [inline vite-sandbox/index.html:14]: Should we consider re-adding the `latest` folder to our CDN?

File: vite-sandbox/src/components/App.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { useState, Fragment } from "react";
+
+const App = () => {
+  const [count, setCount] = useState(0);
+
+  return (
+    <Fragment>
+      <header>
+        <h1>Hello Vite + React!</h1>
```
Comment (jfkhoury) [inline vite-sandbox/src/components/App.jsx:9]: Update title

**Review by jfkhoury — CHANGES_REQUESTED** [review]
Couple of comments and let's please add new lines everywhere.
Thanks.

**Comment by jonsnyder** [comment]
This project is on the back-burner for now.

**Pull request closed without merging**